bazel/google_cloud_cpp_deps.bzl

```diff
@@ -186,3 +186,14 @@ def google_cloud_cpp_deps():
             patch_args = ["-p1"],
             patches = [Label("//bazel:configure_template.bzl.patch")],
         )
+
+    # Open Telemetry
+    if "io_opentelemetry_cpp" not in native.existing_rules():
```

Comment:
The renovate configuration will also need a PR, one where both CMake and Bazel get changed in a single PR.

Reply:
Ack. I added this as an extra task to the GH issue.